web/gulpfile.babel.js

```diff
@@ -37,6 +38,13 @@ gulp.task('test', (done) => {
 
 gulp.task('server-hot', bg('node', './webpack/server'));
 
-gulp.task('server', ['set-dev-environment', 'server-hot'], bg('./node_modules/.bin/nodemon', './src/server'));
+gulp.task('nodemon', () => {
+  nodemon({
+    script: './src/server',
+    env: {'NODE_ENV': process.env.NODE_ENV || 'development'}
```

Comment:
@grabbou I tried to remove it at first, but then it didn't work. I tried it now and it works. So lets remove it :)